openstack/utils/templates/_hosts.tpl

```diff
@@ -46,7 +52,13 @@ postgresql+psycopg2://{{$user}}:{{$password | urlquery}}@{{.Chart.Name}}-postgre
 
 {{- define "db_url_mysql" }}
     {{- if kindIs "map" . }}
-        {{- tuple . (coalesce .Values.dbName .Values.db_name) (coalesce .Values.dbUser .Values.global.dbUser "root") (coalesce .Values.dbPassword .Values.global.dbPassword .Values.mariadb.root_password | required ".Values.mariadb.root_password is required!") .Values.mariadb.name | include "db_url_mysql" }}
+        {{- if and .Values.mariadb.users .Values.mariadb.databases }}
+            {{- $db := first .Values.mariadb.databases }}
+            {{- $user := get .Values.mariadb.users $db | required (print "User with key " $db " required") }}
+            {{- list . $db $user.name (required (printf "User with key %v requires password" $db) $user.password) | include "db_url_mysql" }}
```

Comment:
Why do we switch from `tuple` to `list` in this call vs. the previous/else call?

Reply:
No particular reason. I had lists in my previous change... I'll change it to tuple for consistency